nautobot/docs/user-guide/platform-functionality/dynamicgroup.md

```diff
@@ -2,69 +2,107 @@
 
-+/- 1.4.0
++/- 1.4.0 "Change in empty filter behavior"
     In Nautobot v1.3.0 the default for a Dynamic Group with an empty filter was to "fail closed" and have zero members.
 
     As of v1.4.0, this behavior has been inverted to default to include all objects matching the Content Type, instead of matching no objects as was previously the case. This was necessary to implement the progressive layering of child filters similarly to how we use filters to reduce desired objects from basic list view filters. This will be described in more detail below.
 
-#### Basic Filtering
-
-When editing a Dynamic Group, under the **Filter Options** section, you will find a **Filter Fields** tab that allows one to specify filter criteria. The filter fields available for a given Content Type are backed and validated by underlying filterset classes (for example `nautobot.dcim.filters.DeviceFilterSet`) and are represented in the web interface as a dynamically-generated filter form that corresponds to each eligible filter field.
+When editing a filter-based Dynamic Group, under the **Filter Options** section, you will find a **Filter Fields** tab that allows you to select filter criteria. The filter fields available for a given Content Type are backed and validated by underlying FilterSet classes (for example `nautobot.dcim.filters.DeviceFilterSet`) and are represented in the web interface as a dynamically-generated filter form that corresponds to each eligible filter field.
 
-#### Advanced Filtering
+### Set-Based Dynamic Groups
 
 +++ 1.4.0
 
-Advanced filtering is performed using nested Dynamic Group memberships.
+Set-based Dynamic Groups do not directly define a filter for identifying member objects; instead, they define their members based on set operations (AND, OR, and NOT, or Union, Intersection, and Difference if you prefer) involving _other_ Dynamic Groups.
 
-An object is considered a member of an advanced Dynamic Group if it matches the aggregated filter criteria across all descendant groups.
+!!! tip
+    At this time, a set-based Dynamic Group may include filter-based Dynamic Groups and other set-based Dynamic Groups in its members definition, but set-based Dynamic Groups _may not_ currently include static-assignment-based Dynamic Groups. This restriction may be removed in a future Nautobot release.
 
-When editing a Dynamic Group, under the **Filter Options** section, you will find a **Child Groups** tab that allows one to make other Dynamic Groups of the same Content Type children of the parent group.
+An object is considered a member of an set-based Dynamic Group if it matches the aggregated filter criteria derived from the "descendant" groups included in this group definition by way of the "Dynamic Group Membership" through table.
```

Comment:
```suggestion
An object is considered a member of a set-based Dynamic Group if it matches the aggregated filter criteria derived from the "descendant" groups included in this group definition by way of the "Dynamic Group Membership" through table.
```

Reply:
What is the "Dynamic Group Membership" through table?

Reply:
That's the (poorly named) `DynamicGroupMembership` model. I'll see if I can reword this too.

Reply:
Actually, does the `info` admonition immediately below this line make things clear enough as is?

Reply:
Yeah I think that's sufficient. Is that a new model that you added or did DynamicGroup always have that for set-based groups?

Reply:
That's how it's already been.

Reply:
It might make this easier to understand if you mention that the `extras.DynamicGroupMembership` through table is used for the `DynamicGroup.children` field.

Reply:
Updated hopefully for clarity.